app/about_us/page.js

```diff
@@ -22,7 +24,17 @@ const OurCrew = () => {
   // Use the pictures from /public/crew
   // Some inspiration ideas found in /data/inspiration_about_us
   return (
-    <p> ADD OUR CREW HERE </p>
+   <div>
+    <p>Meet the best space travelers and scientist in the world.</p>
+    <img src="/crew/image-anousheh-ansari.png" alt="Anousheh Ansari" />
```

Comment:
You are missing the width and height properties from the img (either as a html prop here, or from css), that is why you have those huge images 😄